workspace_helpers.bzl

```diff
@@ -239,8 +239,8 @@ erlang_lib(
     github_bazel_erlang_lib(
         name = "seshat",
         org = "rabbitmq",
-        ref = "main",
-        version = "main",
+        ref = "refs/heads/valid-prometheus-format",
+        version = "refs/heads/valid-prometheus-format",
```

Comment:
The value will be injected into `ebin/seshat.app` so I would probably use `version = "valid-prometheus-format"`. I have been considering some enhancements to `github_bazel_erlang_lib` where the version would be derived automatically unless overridden. It would eliminate the need for this line in this case, which would be nice.